cmd/escp/main.go

```diff
@@ -14,16 +14,37 @@ import (
 
 func main() {
 	flag.Usage = func() {
-		fmt.Fprintf(os.Stderr, "Usage of %s http://HOST1:9200/INDEX1 http://HOST2:9200/INDEX2\n", os.Args[0])
+		fmt.Fprintf(os.Stderr, "Usage of %s http://HOST1:9200/INDEX1 HOST2:9200,HOST3:9200,HOST4:9200 INDEX2\n", os.Args[0])
 		flag.PrintDefaults()
 	}
+
+	// Index creation settings
 	shards := 0
 	flag.IntVar(&shards, "shards", shards, "number of shards target index will have (default = same as old index)")
 	skipcreate := false
 	flag.BoolVar(&skipcreate, "skipcreate", skipcreate, "skip destination index creation")
+
+	// Tunables
+	scrolltimeout := "15m"
+	flag.StringVar(&scrolltimeout, "scrolltime", scrolltimeout, "time to keep scroll alive between requests")
+	scrollpage := 1000
+	flag.IntVar(&scrollpage, "scrollpage", scrollpage, "size of scroll pages (will actually be per source shard)")
+	scrolldocs := 1000
+	flag.IntVar(&scrolldocs, "scrolldocs", scrolldocs, "number of `docs` to buffer in memory from scroll")
+	bulksz := 20 * 1024
+	flag.IntVar(&bulksz, "bulksz", bulksz, "size of bulk upload buffer in `KB`")
+	bulkpar := 0
+	flag.IntVar(&bulkpar, "bulkpar", bulkpar, "number of parallel bulk upload buffers to use; 0 = len(hosts)*2")
+	delayrefresh := true
+	flag.BoolVar(&delayrefresh, "delayrefresh", delayrefresh, "delay refresh until bulk indexing is complete")
+	refreshint := "1s"
```

Comment:
k, implemented it so that:

* if refreshint is set, apply it to the new index
* if refreshint isunset but refresh_interval was set on the old index, copy it to the new index
* if refreshint and refresh interval were unset, leave it unset in the new index (so it uses the system default)